ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java

```diff
@@ -115,6 +115,7 @@
     private CCDHearingDurationType estimatedHearingDuration;
 
     private CCDDocument draftOrderDoc;
+    private CCDDocument draftLetterDoc;
```

Comment:
if the letter can be issued multiple times, we cant have it at the CCDCase level